### PR TITLE
feat(clean) : precompile Levenstein and python-crfsuite when release

### PR DESCRIPTION
**Pull request by qidanrui**
# Description
Change the release.yml in workflow to compile Levenstein and python-crfsuite before releasing on different platforms:
OS including: ubuntu-latest, macos-latest and windows-latest
Python version including: 3.8, 3.9

# How Has This Been Tested?
Workflows except create release notes in  `upload` part are passed through, see:
https://github.com/sfu-db/dataprep/actions/runs/2231296563

After install wheel, `clean_phone` and `clean_address` can still run in the normal way

# Snapshots:

Screen Shot 2022-04-26 at 11 46 05 PM

Screen Shot 2022-04-26 at 11 45 47 PM

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have already squashed the commits and make the commit message conform to the project standard.
- [x] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,11 +3,13 @@ name: CI
 on:
   push:
     branches:
-      - develop
```
Comment (dovahcrow) [inline .github/workflows/ci.yml]: here

File: dataprep/clean/address_utils.py [hunk 2]
```diff
@@ -1318,6 +1316,7 @@ def load_model() -> Any:
     Load address parsing model from local directory.
     """
     tagger = None
+    print(pycrfsuite)
```
Comment (dovahcrow) [inline dataprep/clean/address_utils.py]: here

File: dataprep/tests/clean/test_clean_address.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 import pytest
 
-from ...clean import clean_address, validate_address
+from dataprep.clean import clean_address, validate_address
```
Comment (dovahcrow) [inline dataprep/tests/clean/test_clean_address.py]: here

File: pyproject.toml [hunk 4]
```diff
@@ -58,11 +63,14 @@ tqdm = "^4.48"
 flask = "^2"
 flask_cors = "^3.0.10"
 metaphone = "^0.6"
-python-Levenshtein = "^0.12.2"
-python-crfsuite = "^0.9.7"
 python-stdnum = "^1.16"
 regex = "^2021.8.3"
 varname = "^0.8.1"
+scikit-build = "^0.14.1"
```
Comment (dovahcrow) [inline pyproject.toml]: all these are runtime dependencies?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (qidanrui) [reply .github/workflows/ci.yml]: Shouldn't we trigger the ci workflow when pushing to develop branch?

**Pull request merged**